### PR TITLE
Changed module name and make minor edits

### PR DESCRIPTION
**Pull request by JoanneOFlynn2018**

### CODE REVIEW

**Comment by JoanneOFlynn2018** [comment]
I edited this and changed the heading.
Because this is a collectd plugin, the format and location might change in the future. Leaving it here for now.

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc [hunk 1]
```diff
@@ -23,23 +23,24 @@
...
-[id="service-resource-usage-metrics_{context}"]
-= OpenStack service monitoring
-Conceptual overview:
-Monitoring the resource usage of the openstack services such as the APIs and other infrastructure processes can be very useful in identifying areas of the cloud that are under a lot of stress. This can be done by enabling the collectd-libpod-stats plugin to gather CPU and memory usage metrics for every container running in the overcloud.
+[id="monitoring-resource-usage-of-openstack-services_{context}"]
+= Monitoring the resource usage of {OpenStack} services
 
-Requirements:
-{OpenStackVersion} //Train and OSP16.1.3
+Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify areas of the cloud that are under a lot of stress. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc:29]: "to identify areas of the cloud that are under a lot of stress."

@pleimer Can you be more specific here? For example, to identify areas that need more compute power, something like that.

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc [hunk 2]
```diff
@@ -23,23 +23,24 @@
...
----
-parameter_defaults:
-  ...
   CollectdEnableLibpodstats: true
-
 ```
-- Continue with the overcloud deployment procedure
\ No newline at end of file
+
+. Continue with the overcloud deployment procedure.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc:46]: For future editing.

**Comment by leifmadsen** [comment]
I _think_ this will merge clean with the other PR I just opened, but if not, I will deal with the merge conflicts and get everything in to shape.

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc (hunk 1 above)
Comment (pleimer) [reply doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc:29]: That probably is the best clarification:
"to identify areas that need more compute power"

Comment (pleimer) [reply doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc:29]: If we want to include more details, we could add a phrase like: "and identify bottlenecks in the cloud"

Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc:29]:
```suggestion
Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify areas of the cloud that require more compute power. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.
```

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc [hunk 3]
```diff
@@ -23,23 +23,24 @@
...
+.Prerequisites
 
-- Modify your `stf-connectors.yaml`. See xref:configuring-red-hat-openstack-platform-overcloud-for-stf[] for more information about creating `stf-connectors.yaml`.
-- Add `CollectdEnableLibpodstats: true`
+* You have created the `stf-connectors.yaml` file. For more information, see xref:configuring-red-hat-openstack-platform-overcloud-for-stf_completing-the-stf-configuration[].
+* You are using the most current version of {OpenStack}: {OpenStackVersion}.
+
+.Procedure
+
+. Open the `stf-connectors.yaml`.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc:38]:
```suggestion
. Open the `stf-connectors.yaml` file.
```

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc (hunk 1 above)
Comment (pleimer) [reply doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc]: "to identify bottlenecks in the overcloud by showing services running out of compute power" or something like that

File: doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc [hunk 4]
```diff
@@ -23,23 +23,24 @@
...
-[id="service-resource-usage-metrics_{context}"]
-= OpenStack service monitoring
-Conceptual overview:
-Monitoring the resource usage of the openstack services such as the APIs and other infrastructure processes can be very useful in identifying areas of the cloud that are under a lot of stress. This can be done by enabling the collectd-libpod-stats plugin to gather CPU and memory usage metrics for every container running in the overcloud.
+[id="monitoring-resource-usage-of-openstack-services_{context}"]
+= Monitoring the resource usage of {OpenStack} services
 
-Requirements:
-{OpenStackVersion} //Train and OSP16.1.3
+Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify areas of the cloud that require more compute power. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_monitoring-resource-usage-of-openstack-services.adoc:29]:
```suggestion
Monitor the resource usage of the {OpenStack} services, such as the APIs and other infrastructure processes, to identify bottlenecks in the overcloud by showing services running out of compute power. Enable the `collectd-libpod-stats` plug-in to gather CPU and memory usage metrics for every container running in the overcloud.
```

**Pull request merged**